<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_14" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="btn_back">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/icons8-back-26.png</normaloff>../img/icons8-back-26.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../img/icons8-animal-44.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../img/icons8-multiply-26.png</normaloff>../img/icons8-multiply-26.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_15" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">color: #1b1209;
font: 18pt &quot;Bahnschrift&quot;;</string>
           </property>
           <property name="text">
            <string>Where would you like to search?</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_16" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>50</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">/* Thiết lập CSS cho combobox */
QComboBox {
    color: #575556;
    font: 14pt &quot;Bahnschrift&quot;;
    border-style: solid;
    border-width: 2px;
    border-color: #ECE7E0;
    padding-left: 10px;
    border-radius: 15px;
    background-color: rgba(255, 0, 0, 0); /* Màu nền sẽ không ảnh hưởng do transparency (alpha = 0) */
}

/* Thiết lập màu nền cho các mục trong combobox khi chưa hover */
QComboBox::item {
    background-color: #575556;
	border-radius: 0px;
}

/* Thiết lập màu nền cho các mục trong combobox khi hover */
QComboBox::item:hover {
    background-color: #ECE7E0;
	border-radius: 0px;
}

QComboBox::item:selected {
    background-color: #ECE7E0;
	border-radius: 0px;
}
</string>
           </property>
           <property name="currentText">
            <string/>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
           <property name="placeholderText">
            <string>Dog breed</string>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Bulldog</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pug</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Shiba</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Chihuahua</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Husky</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>50</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">/* Thiết lập CSS cho combobox */
QComboBox {
    color: #575556;
    font: 14pt &quot;Bahnschrift&quot;;
    border-style: solid;
    border-width: 2px;
    border-color: #ECE7E0;
    padding-left: 10px;
    border-radius: 15px;
    background-color: rgba(255, 0, 0, 0); /* Màu nền sẽ không ảnh hưởng do transparency (alpha = 0) */
}

/* Thiết lập màu nền cho các mục trong combobox khi chưa hover */
QComboBox::item {
    background-color: #575556;
	border-radius: 0px;
}

/* Thiết lập màu nền cho các mục trong combobox khi hover */
QComboBox::item:hover {
    background-color: #ECE7E0;
	border-radius: 0px;
}

QComboBox::item:selected {
    background-color: #ECE7E0;
	border-radius: 0px;
}
</string>
           </property>
           <property name="placeholderText">
            <string>Dog color</string>
           </property>
           <item>
            <property name="text">
             <string>Black</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Grey</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>White</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Brown</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tabbly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Other</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_3">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">/* Thiết lập CSS cho combobox */
QComboBox {
    color: #575556;
    font: 14pt &quot;Bahnschrift&quot;;
    border-style: solid;
    border-width: 2px;
    border-color: #ECE7E0;
    padding-left: 10px;
    border-radius: 15px;
    background-color: rgba(255, 0, 0, 0); /* Màu nền sẽ không ảnh hưởng do transparency (alpha = 0) */
}

/* Thiết lập màu nền cho các mục trong combobox khi chưa hover */
QComboBox::item {
    background-color: #575556;
	border-radius: 0px;
}

/* Thiết lập màu nền cho các mục trong combobox khi hover */
QComboBox::item:hover {
    background-color: #ECE7E0;
	border-radius: 0px;
}

QComboBox::item:selected {
    background-color: #ECE7E0;
	border-radius: 0px;
}
</string>
           </property>
           <property name="placeholderText">
            <string>Gender</string>
           </property>
           <item>
            <property name="text">
             <string>Male</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Female</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btn_continue">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton 
{
background-color: #356a9c;
color: #575556;
font: 16pt &quot;Bahnschrift&quot;;
 border-style: solid;
 border-width: 2px;
border-color: #ECE7E0;
border-radius: 15px;
color: white;
}
QPushButton:hover
{
background-color: #14316c;
}</string>
           </property>
           <property name="text">
            <string>Continue</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
